README.md

```diff
@@ -119,6 +129,8 @@ To enable the Staking page and menu item, set the following variable to *true* i
 VUE_APP_ENABLE_STAKING=true
 ```
 
+### Customize configuration
+
```

Comment:
Missing content?

Reply:
No, the diff is a bit misleading because we have adjusted the structure of the README between the creation of this PR and now.